__tests__/integration.js

```diff
@@ -103,6 +103,50 @@ describe.each(EACH_MATRIX)('%s:%s: integration tests', (eventSourceName, framewo
     expect(response).toEqual(expectedResponse)
   })
 
+  test('GET Uint8Array', async () => {
+    const multiValueQueryStringParameters = {
+      singleNormal: ['1']
+    }
+    const queryStringParameters = {
+      singleNormal: '1'
+    }
+    router.get('/uint8', (req, res) => {
+      const { singleNormal } = req.query
+      res.set('X-Custom-Header', singleNormal)
+
+      const readStream = new stream.PassThrough()
+      const buffer = Uint8Array.from(
+        Array.from('Hello serverless!').map(ch => ch.charCodeAt(0))
+      )
+
+      res.set('content-length', Buffer.byteLength(buffer))
+      res.set('content-type', 'application/octect-stream')
```

Comment:
Typo, should say `application/octet-stream` :-)

Reply:
Thanks for pointing it out!